intro-HPC/ch_MATLAB.tex

```diff
@@ -79,15 +81,36 @@ \subsection{Memory issues during compilation}
 if it's trying to create to many Java processes. If we lower the heap size, more
 Java processes will be able to fit in memory.
 
+Another possible issue is that the heap size is too small. This could result
+in errors like:
+
+\begin{prompt}
+Error: Out of memory
+
+Typically, this message occurs because the compiler requests a larger segment of memory from the operating system than is currently available. Adding additional memory to your system can alleviate this problem.
```

Comment:
`Out of memory` is the error, the rest is an answer from the MATLAB website, we shouldn't by copy-pasting that :)